content/root/best-practice-guide.textile

```diff
@@ -0,0 +1,146 @@
+
+This guide consists of the following two sections:
+* Ably 101 - Key concepts that you must understand before moving on to the best practice recommendations.
+* Using Ably like a pro - Actual recommendations that we've compiled as per our experience with our users, in order for you to avoid common mistakes or misunderstandings whilst using Ably.
+
+h3(#ably-101). Ably 101
+
+Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play.
+
+h4(#pubsub). Publish and Subscribe (Pub/Sub)
+
+You can publish data to a named "channel":/realtime/channels-messages on the Ably Realtime platform, and all the clients subscribed to that very channel will receive this data in realtime, as it is being published.
+
+h4(#notifications). Notifications 
+
+You can use "Push Notifications":/realtime/push to send notifications to your clients even when they are currently not using your application, or in other words, not connected to Ably. You can do this on mobile devices as well as browsers. You can also use "Smart Notifications":/general/smart-notifications for your notifications to be most effective depending on your user's connection statte, their active device or even their current context within your application.
+
+h4(#presence). Presence
+
+You can have your clients attach to the presence channel using "Presence":/realtime/presence in order to indicate if these clients (or devices in case of IoT clients) are online or connected. The changes in this connection status, if any, can also be tracked by optionally having your clients subscribe to the same presence channel.
+
+h4(#rest-or-realtime). Realtime vs REST
+
+With the availability of both the "Realtime":/realtime and "REST":/rest libraries, choose whichever suits your application's needs the best and get started. Using the realtime library, your clients can subscribe to certain data that Ably can then push to your clients as it is being created. On the other hand, the REST library offers simple stateless data publish or retrieval to and from Ably respectively.
+
+h4(#auth). Choose the right auth scheme
+
+You can use "Basic Authentication":/general/authentication#basic-authentication to easily authenticate your clients using an API key or use "Token Authentication":/general/authentication#token-authentication to authenticate your clients using a token request sent to Ably via your auth server. Furthermore, you can use JWT authentication as part of Tokent Authentication ensuring that you use a single authentication strategy for all of your third party services. You can also "encrypt your messages":/realtime/encryption for added security of your application. 
```

Comment:
Tokent typo.  I don't think this explains why someone should use one auth scheme over another,  It does not explain how tokens work, and then it introduces encryption which I previously mentioned has nothing to do with authentication,.